docs/Projects/TAD/Reporting-Standard/0.1a2.md

```diff
@@ -0,0 +1,407 @@
+18. `external_providers` (OPTIONAL, list[string]). Name of an external provider, if relevant. There can
+be multiple external providers.
+19. `references` (OPTIONAL, list[string]). Additional reference URI's that point information about the system
+and are relevant.
+
+#### 1. Models
+
+1. `models` (OPTIONAL, list[ModelCard]). A list of model cards (as defined below) or `!include`s of a yaml
+file containing a model card. This model card can for example be a model card described in the next section
+or a model card from HuggingFace. There can be multiple model cards, meaning multiple models are used.
+
+#### 2. Assessments
+
+1. `assessments` (OPTIONAL, list[AssesmentCard]). A list of assessment cards (as defined below) or `!include`s of a yaml
+file containing a assessment card. This assessment card is an assessment card described in the next section.
+There can be multiple assessment cards, meaning multiple assessment were performed.
+
+### `model_card`
+
+A `model_card` contains the following information.
+
+1. `language` (OPTIONAL, list[string]). If relevant, the natural languages the model supports in [ISO 639](https://www.iso.org/iso-639-language-code).
+    There can be multiple languages.
+2. `license`(REQUIRED, string). Any license from the [open source license list](https://opensource.org/license)
+[^1]. If the license is NOT present in the license list this field must be set to 'other' and the following
+two fields will be REQUIRED.
+
+    1. `license_name` (string). An id for the license.
+    2. `license_link` (string). A link to a file of that name inside the repo, or a URL to a remote file containing the license
+    contents.
+
+3. `tags` (OPTIONAL, list[string]). Tags with keywords to describe the project. There can be multiple tags.
+4. `owners` (list). There can be multiple owners. For each owner the following fields are present.
+
+    1. `oin` (OPTIONAL, string). If applicable the [Organisatie-identificatienummer (OIN)](https://oinregister.logius.nl/oin-register).
+    2. `organization` (OPTIONAL, string). Name of the organization that owns the model. If `ion` is
+    NOT provided this field is REQUIRED.
+    3. `name` (OPTIONAL, string). Name of a contact person within the organisation.
+    4. `email` (OPTIONAL, string). Email address of the contact person or organization.
+    5. `role` (OPTIONAL, string). Role of the contact person. This field should only be set when the `name` field
+    is set.
+
+#### 1. Model Index
+
+There can be multiple models. For each model the following fields are present.
+
+1. `name` (REQUIRED, string). The name of the model.
+2. `model` (REQUIRED, string). A URI pointing to a repository containing the model file.
+3. `artifacts` (OPTIONAL, list). A list of artifacts
+
+    1. `uri` (OPTIONAL, string) URI refers to a relevant model artifact
+    2. `type` (OPTIONAL, string) Optional type, recognized values are "onnx", to refer to an ONNX representation of the model.
```

Comment:
@ChristopherSpelt @robbertbos this is the actual change I'm proposing: going from a flat list to this

Reply:
@anneschuth I agree with adding more (optional) info about artifacts. I have some suggestions and curious what you think of them:

1. `type` -> `concent-type`: maybe we can follow the [Content-Type](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type) as described by [MDN Web Docs](https://developer.mozilla.org/).
2. add more file related info, f.e. `size` and `etag` (f.e. MD5 digest)
3. in case of zip archive, add info about files & directories in the zip archive

Reply:
I've switched to `content-type`, and added `md5-checksum`

Reply:
(in this new format, we can extend as we wish and see use)